### DISCUSSION TITLE
Are there plans to bring the Form primitive to radix-ui/themes?

### DISCUSSION THREAD

**Discussion by damassi** [radix-ui/themes#650] [Ideas]
Excellent library! Just noticed that there's a TextField, but no Form. Given the importance of forms, Its hard to make this consistent with everything else using custom CSS. Are there plans to add this to the lib?

**Comment by chaance** [comment]
At some point, perhaps! We've got some other features/updates we're prioritizing at the moment. Gonna move this over into a discussion for future consideration.
